automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookPyKernelSpec.scala

```diff
@@ -38,76 +41,9 @@ class NotebookPyKernelSpec extends RuntimeFixtureSpec with NotebookTestUtils {
         }
     }
 
-    "should include Content-Security-Policy in headers" in { runtimeFixture =>
-      val headers = Notebook.getApiHeaders(runtimeFixture.runtime.googleProject, runtimeFixture.runtime.clusterName)
-      val contentSecurityHeader = headers.find(_.name == "Content-Security-Policy")
-      contentSecurityHeader shouldBe defined
-      contentSecurityHeader.get.value should include("https://bvdp-saturn-dev.appspot.com")
-      contentSecurityHeader.get.value should not include "https://bvdp-saturn-prod.appspot.com"
-      contentSecurityHeader.get.value should not include "*.terra.bio"
-    }
-
-    "should allow BigQuerying via the command line" in { runtimeFixture =>
-      withWebDriver { implicit driver =>
-        withNewNotebook(runtimeFixture.runtime) { notebookPage =>
-          val query =
-            """! bq query --format=json "SELECT COUNT(*) AS scullion_count FROM [bigquery-public-data.samples.shakespeare] WHERE word='scullion'" """
-          val expectedResult = """[{"scullion_count":"2"}]""".stripMargin
-
-          val result = notebookPage.executeCell(query, timeout = 5.minutes).get
-          result should include(expectedResult)
-        }
-      }
-    }
+    // TODO: [discuss] it does not seem like automation tests should be verifying security policies
 
-    "should update dateAccessed if the notebook is open" in { runtimeFixture =>
```

Comment:
I feel like we should keep this since we rely on dateAccessed to be successfully updated to do our autopause. Maybe we could put the spleep back to 90, or 60 seconds instead of 120?

Reply:
I think this is fairly unit testable as well. It's essentially testing 'when i hit this api, this database field is updated'

This isn't even using the same API that terra-ui would be hitting to trigger the date accessed update, that would be the proxy traffic. (We only list passively when polling, we never `get` unless you are looking at the compute modal details).

Reply:
Same comment as above then, seems like you uncovered a pattern of integration tests that should be unit tests instead